docs/SNAPTEL.md

```diff
@@ -72,9 +78,10 @@ watch        watch <task_id>
 enable       enable <task_id>
 help, h      Shows a list of commands or help for one command
 ```
+
 #### plugin
 ```
-$ $SNAP_PATH/bin/snaptel plugin command [command options] [arguments...]
+$ snaptel plugin command [command options] [arguments...]
```

Comment:
Please, update commands for snaptel plugin (unchanged fragment below), especially for `snaptel plugin unload` (it's related to my next comment)